### PR TITLE
CMake: Download PDF manual from downloads.mixxx.org

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

**Comment by Holzhaus** [comment]
CI fails because the PDFs do not exist on the server. https://github.com/mixxxdj/manual/pull/367 needs to be merged first.

File: CMakeLists.txt [hunk 1]
```diff
@@ -1616,6 +1623,26 @@ add_dependencies(mixxx-test mixxx-testdata)
 # library. Doing this would require initialization using Q_INIT_RESOURCE()
 # calls that are not present at the moment. Further information can be found
 # at: https://doc.qt.io/qt5/resources.html#using-resources-in-a-library
+option(DOWNLOAD_MANUAL_PDF "Download Manual PDF from Mixxx website" ON)
```
Comment (Be-ing) [inline CMakeLists.txt:1626]: Do we really want this on by default?

Comment (Holzhaus) [reply CMakeLists.txt:1626]: I don't care. Before, the manual was included so I enabled it by default. The download is ~30MiB and we do pull other dependencies like libkeyfinder by default.

Comment (Be-ing) [reply CMakeLists.txt:1626]: I think it should be off by default. I don't care about downloading the manual PDF when building locally. This matters for packaging, not really for local builds.

Comment (Holzhaus) [reply CMakeLists.txt]: Ok, changed.

**Comment by Holzhaus** [comment]
This works now because the PDFs have been uploaded to downloads.mixxx.org manually. We should still sort out the CI deployment though, but that is unrelated to this branch.

**Comment by Holzhaus** [comment]
Merge? This will make sure we'll always have up-to-date PDFs in Mixxx (as long as the PDFs on downloads.mixxx.org are up-to-date ofc).

**Comment by ronso0** [comment]
Thanks LGTM

**Pull request merged**